### ISSUE TITLE
Is Bet365 detecting it?

### ISSUE THREAD

**Issue by espizan99** [ultrafunkamsterdam/undetected-chromedriver#283]
Hi, 

First of all thank you for your work, it is impressive.

I have been testing it with my bet365 account and the thing is when I log in with my bot calling undetected-chromedriver bet365 says that my credentials are wrong, but if I try to connet it without any python bot (manual mode) everything is right!

I have tried it with many several ips and several users but no luck.

Does anyone experienced the same problem recently?

@ultrafunkamsterdam could you take it a look, please?

Thanks and best regards,

**Comment by espizan99** [comment]
Problem solved

**Comment by ENUM1** [comment]
@espizan99 what was the solution?

**Comment by espizan99** [comment]
Hi @ENUM1,

Are you experiencing the same problem?

It is not totally solved, because it still happens nowadays.

I think that bet365 detects it because when I join manually there is no problem.

Anybody is suffering it?

**Comment by ENUM1** [comment]
Yes @espizan99 - I would say that currently I am blocked by bet365 on login about 80% of the time. 

The response is that "login details are not recognised".

**Comment by espizan99** [comment]
Yes, same problem here @ENUM1 .
What I realized is that changing ip it works for the first login but the second login is impossible. 
What I did to my code was loading my chrome profile with selenium like this:

        options = webdriver.ChromeOptions()
        options.add_argument('--user-data-dir=' + str(Path.home()) + '\\AppData\\Local\\Google\\Chrome\\User Data')
        options.add_argument('--profile-directory=' + 'Default')
        uc.TARGET_VERSION = 91
        self.driver = uc.Chrome(options=options)

And only login once and keep the session alive (refreshing, navigating and things like that), but sometimes that I need to stop the process and I need to login again it can´t (and sometimes it is like that they log me out and I can´t login neither).

This is like a workarround that stills fails, but they are detecting the bot (manually always work).

Could anyone help us with this? Or anyone experiencing this?

Thanks and best regards,

**Comment by rwguerra** [comment]
Hi guys! Im developing something on bet365... do you want to create a telegram group or something so we can discuss together? telegram: @guerraricardo
